### PR TITLE
Remove Paros Reports

### PR DESCRIPTION
**Pull request by psiinon**
Part of #6483

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Review by kingthorin — APPROVED** [review]

File: zap/zap.gradle.kts [hunk 1]
```diff
@@ -149,6 +149,7 @@ val japicmp by tasks.registering(JapicmpTask::class) {
     packageExcludes = listOf(
         // Deprecated in 2.10, removal part of 2.11 milestone
         "org.parosproxy.paros.extension.encoder",
+        "org.parosproxy.paros.extension.report",
         "org.zaproxy.zap.extension.encoder2")
```
Comment (kingthorin) [inline zap/zap.gradle.kts:154]: Not a blocker but maybe but report at the bottom? I believe the comment at 150 applies to both encoder lines doesn't it?

Comment (psiinon) [reply zap/zap.gradle.kts]: Done

**Comment by thc202** [comment]
We should also remove the resource messages no longer needed.

**Review by kingthorin — APPROVED** [review]

**Comment by psiinon** [comment]
Just noticed I'm getting an error: `java.util.MissingResourceException: Can't find resource for bundle java.util.PropertyResourceBundle, key menu.report.html.generate`
Looks like this resource is being used by org.zaproxy.zap.extension.customreport.ExtensionCustomReport :/
So I'll need to add that back in or change and rerelease that add-on :/

**Comment by thc202** [comment]
Or remove the add-on from the marketplace (in dev and then following version), that's one of the superseded add-ons.

**Comment by psiinon** [comment]
I've put that label back in. I agree the other reports should be removed from the marketplace, but thats a separate issue / action.

**Comment by thc202** [comment]
Is the removal of the resource message being tracked in the referenced issue?

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Comment by psiinon** [comment]
> Is the removal of the resource message being tracked in the referenced issue?

It is now :D

**Comment by psiinon** [comment]
Yes :)

**Comment by thc202** [comment]
Thank you.

**Pull request merged**